.github/workflows/build.yml

```diff
@@ -4,6 +4,8 @@ on:
   pull_request:
     paths-ignore: ['**.md']
   push:
+    branches:
+      - develop
```

Comment:
Without this here itll run the build/unit tests twice at the same time for pr pushes